src/Events/Models/CloudEventEnvelope.cs

```diff
@@ -13,7 +10,7 @@ public class CloudEventEnvelope
         /// <summary>
         /// The Event to push
         /// </summary>
-        public CloudEvent CloudEvent { get; set; }
+        public object CloudEvent { get; set; }
```

Comment:
I don't recall, did we find a good way to test this change?

Reply:
I tried testing last night, but didn't get very far.  My suggestion would be that we keep cloudEvent as the type for now, and rather update this in the issues related to pushing external events. We don't have support for generating subscriptions, so no cloud envelope will be generated for external events with potential additional params yet.